README.md

```diff
@@ -12,7 +12,11 @@ This component allows you to process given _CSV_ files in order to generate SPHE
 
 ```bash
 $ npm install -g sphere-product-type-json-generator
-
+```
+* build javascript sources
```

Comment:
i think this should be in a seperate section called "Development". only in this case you will need to build the js files